packages/react/src/internal/visible/Visible.tsx

```diff
@@ -0,0 +1,27 @@
+import classNames from 'classnames';
```

Comment:
I wonder if we could apply our own classNames util (https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/utils/classNames.ts) instead of adding this new library?